### PR TITLE
Feature/tinymce5update 3 releases 5.2.2 to 5.2

### PR DESCRIPTION
**Pull request by lorenzo-pomili**
Related Ticket: 

Description of Changes, it covers:

- Release TinyMCE 5.2.2
- Release TinyMCE 5.2.1
- Release TinyMCE 5.2

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/partials/misc/releasenotes_for_stable.adoc [hunk 1]
```diff
@@ -1 +1 @@
-NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: [TinyMCE Changelog](https://www.tiny.cloud/docs/changelog/).
+NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: link:https://www.tiny.cloud/docs/changelog/[TinyMCE Changelog].
```
Comment (lnewson) [inline modules/ROOT/partials/misc/releasenotes_for_stable.adoc:1]:
```suggestion
NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: xref:changelog.adoc[TinyMCE Changelog].
```

File: modules/ROOT/pages/release-notes/release-notes52.adoc [hunk 2]
```diff
@@ -25,37 +26,38 @@ The following new features and enhancements were added for the {productname} 5.2
 
 The `toolbar_location` option is used to position the toolbar and menu bar. Setting this option to `bottom` positions the toolbar and menu bar below the content area.
 
-For information on using the `toolbar_location` setting, see: link:{baseurl}/configure/editor-appearance/#toolbar_location[User interface options - `toolbar_location`].
+For information on using the `toolbar_location` setting, see: xref:configure/editor-appearance.adoc#toolbar_location[User interface options - `toolbar_location`].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes52.adoc:29]: This xref doesn't resolve as the `toolbar_location` id doesn't exist on the relevant heading in `toolbar_location.adoc`. e.g:
```
[[toolbar_location]]
== `toolbar_location`
```

File: modules/ROOT/pages/release-notes/release-notes52.adoc [hunk 3]
```diff
@@ -115,31 +118,32 @@ The {productname} 5.2 release includes an accompanying release of the *Accessibi
...
+For information on the `linkchecker_preprocess` option, see: xref:plugins/premium/linkchecker.adoc#linkchecker_preprocess[Link Checker plugin - `linkchecker_preprocess`].
 
 === Page Embed 1.1.0
 
 The {productname} 5.2 release includes an accompanying release of the *Page Embed* premium plugin.
 
 *Page Embed* 1.1.0 adds a new `tiny_pageembed_inline_styles` setting which inlines all styles, while also keeping CSS classes, when rendering the embed content. This is useful when the additional CSS classes can't be added to your site.
 
-For information on inlining page embed styles, see: link:{baseurl}/plugins/premium/pageembed/#inlinestylesforpageembeds[Page Embed plugin - Inline styles for page embeds].
+For information on inlining page embed styles, see: xref:plugins/premium/pageembed.adoc#inlinestylesforpageembeds[Page Embed plugin - Inline styles for page embeds].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes52.adoc:146]: This xref doesn't resolve as the `inlinestylesforpageembeds` id doesn't exist in `pageembed.adoc`. Now this appears to be broken on production as well, so based on the changes done to that file I believe it should be this instead.
```suggestion
For information on inlining page embed styles, see: xref:plugins/premium/pageembed.adoc#tiny_pageembed_inline_styles[Page Embed plugin - Inline styles for page embeds].
```
However, that will also need the id to be added to `tiny_pageembed_inline_styles.adoc`. e.g:
```
[[tiny_pageembed_inline_styles]]
=== `tiny_pageembed_inline_styles`
```

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/partials/misc/releasenotes_for_stable.adoc (hunk 1 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/misc/releasenotes_for_stable.adoc]: done

File: modules/ROOT/pages/release-notes/release-notes52.adoc (hunk 2 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes52.adoc:29]: done

File: modules/ROOT/pages/release-notes/release-notes52.adoc (hunk 3 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes52.adoc]: done

**Review by metricjs — APPROVED** [review]

**Pull request merged**